### ISSUE TITLE
Incompatible with latest Strapi beta

### ISSUE THREAD

**Issue by mbrowne** [Lith/strapi-provider-upload-google-cloud-storage#7]
This provider seems to be incompatible with Stapi 3 beta (tested with 3.0.0-beta.16.7). I got the following error when trying to upload a file (this is the backend error; the response to the frontend was just a 500 internal server error):

```
app_1  | [2019-10-07T14:32:41.799Z] error TypeError: Cannot read property '0' of undefined
app_1  |     at Promise (/usr/src/api/strapi-app/node_modules/strapi-provider-upload-google-cloud-storage/lib/index.js:120:52)
app_1  |     at new Promise (<anonymous>)
app_1  |     at Object.upload (/usr/src/api/strapi-app/node_modules/strapi-provider-upload-google-cloud-storage/lib/index.js:119:24)
app_1  |     at uploadFile (/usr/src/api/strapi-app/node_modules/strapi-plugin-upload/services/Upload.js:77:21)
app_1  |     at Promise.all.files.map.file (/usr/src/api/strapi-app/node_modules/strapi-plugin-upload/services/Upload.js:93:42)
app_1  |     at Array.map (<anonymous>)
app_1  |     at Object.upload (/usr/src/api/strapi-app/node_modules/strapi-plugin-upload/services/Upload.js:93:30)
app_1  |     at Object.upload (/usr/src/api/strapi-app/node_modules/strapi-plugin-upload/controllers/Upload.js:101:47)
app_1  |     at processTicksAndRejections (internal/process/next_tick.js:81:5)
app_1  | [2019-10-07T14:32:41.799Z] debug POST /upload (47 ms) 500
```

I switched to [strapi-provider-upload-google-storage](https://www.npmjs.com/package/strapi-provider-upload-google-storage) and it worked fine.

At first glance at least, it looks like it should be simple to fix—just need to check that `file.related` is actually defined (I don't know if it's ever defined in Strapi beta, I just know that it was undefined when I tried it, which caused the error).

**Comment by Narmdo** [comment]
I'm having the same issue. Exact same error.

**Comment by Lith** [comment]
I'm using the plugin with Strapi 3.0.0-beta.16.8 for now and everything works fine.
Could you please provide us more informations ? 
- Version used of the plugin ?
- Configuration set ?
Thanks !

**Comment by Lith** [comment]
The problem has been fixed in latest version (3.0.0-beta.17) :) 
https://github.com/Lith/strapi-provider-upload-google-cloud-storage/pull/6
Thanks !
